pyproject.toml

```diff
@@ -0,0 +1,226 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "youtube2zim"
+authors = [{ name = "Kiwix", email = "dev@kiwix.org" }]
+keywords = ["kiwix", "zim", "offline", "youtube"]
+requires-python = ">=3.11"
+description = "Make ZIM file from a Youtube channel, user or playlist(s)"
+readme = "README.md"
+license = { text = "GPL-3.0-or-later" }
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+  "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+]
+dependencies = [
+  "python-slugify==3.0.3",
+  "yt-dlp",                      # youtube-dl should be updated as frequently as possible
+  "python-dateutil==2.8.0",
+  "jinja2>=2.11,<3.0",
```

Comment:
Why isn't this pinned?

Reply:
This will be once updated to proper version. I can update Jinja2 for now because it is blocked by zimscraperlib (as far as I remember). It will all be done (update Jinja2 + zimscraperlib) with https://github.com/openzim/youtube/issues/175

Reply:
btw I think scraperlib can be updated easily given we use the zimwriterfs mode (`make_zim_file`) which did not change much.